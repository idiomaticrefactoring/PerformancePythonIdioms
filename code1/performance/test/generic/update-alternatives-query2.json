{"name":"php-fpm.sock","link":"/run/php/php-fpm.sock","status":"auto","best":"/run/php/php8.1-fpm.sock","value":"/run/php/php8.1-fpm.sock","alternatives":[{"alternative":"/run/php/php7.4-fpm.sock","priority":74},{"alternative":"/run/php/php8.0-fpm.sock","priority":80},{"alternative":"/run/php/php8.1-fpm.sock","priority":81}]}
