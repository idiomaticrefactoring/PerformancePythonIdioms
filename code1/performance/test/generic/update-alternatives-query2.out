Name: php-fpm.sock
Link: /run/php/php-fpm.sock
Status: auto
Best: /run/php/php8.1-fpm.sock
Value: /run/php/php8.1-fpm.sock

Alternative: /run/php/php7.4-fpm.sock
Priority: 74

Alternative: /run/php/php8.0-fpm.sock
Priority: 80

Alternative: /run/php/php8.1-fpm.sock
Priority: 81
